### ISSUE TITLE
Generating HTML Coverage for Neko

### ISSUE THREAD

**Issue by ashes999** [massiveinteractive/mcover#37]
TLDR: Is there a way to generate an HTML report against Neko-built code, and see which lines of my classes aren't covered by any tests?

I'm working on [Butterfly](http://github.com/ashes999/butterfly), which is a static website generator; I build against Neko (and only Neko) because I read/write a lot of files from the file system.

I'm at a loss about how to generate a proper HTML coverage report, like EMMA for Java (there's a sample report [here](http://emma.sourceforge.net/coverage_sample_a/index.html)). 

What I really want is some command-line option (or client option) that generates a bunch of HTML files which allows me to drill down class by class and see what lines are not covered at all by any tests.

It doesn't seem like this is possible -- am I mistaken? The current `RichPrintClient` is the best thing so far, and doesn't generate this level of granularity. There's some sort of `test-runner/index.html` file generated, but it doesn't give me anything with my builds (presumably because I'm using Neko, and I may be mixing up MUnit and MCover reports here).
